events/syscall/change_namespace_privillege_using_unshare.go

```diff
@@ -0,0 +1,40 @@
+//go:build linux
+// +build linux
+
+// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright (C) 2024 The Falco Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package syscall
+
+import (
+    "os/exec"
+    "syscall"
+
+    "github.com/falcosecurity/event-generator/events"
+)
+
+var _ = events.Register(ChangeNamespacePrivilegesViaUnshare)
```

Comment:
We should disable by default since the rule is an incubating one:
```
events.WithDisabled(), // this rules is not included in falco_rules.yaml (stable rules), so disable the action
```

See https://github.com/falcosecurity/rules/blob/main/rules/falco-incubating_rules.yaml#L449

Reply:
Now that i double checked, this one too should be disabled by default: https://github.com/falcosecurity/event-generator/pull/153/files

See https://github.com/falcosecurity/rules/blob/main/rules/falco-incubating_rules.yaml#L1263